modules/core/src/main/java/org/apache/ignite/internal/processors/cache/distributed/dht/preloader/GridDhtPartitionsExchangeFuture.java

```diff
@@ -730,6 +731,33 @@ public GridDhtPartitionExchangeId exchangeId() {
         return exchId;
     }
 
+    /**
+     * Returns a collection of partitions (per cache group) that did not pass validation by update counter or size.
+     *
+     * Please take into account that the following cache configurations disable partitions validation:
```

Comment:
Could you please rephrase it?

Reply:
I changed in the following way:
`Partitions are not validated in the following cases and therefore they will not be returned by this method`